reference/hoon-errors.md

```diff
@@ -20,28 +20,28 @@ a correct Hoon file.  These values are always correct.
 Usually, the line and column tell you everything you need to
 know.  But the worst-case scenario for a syntax error is that,
 somewhere above, you've confused Hoon's tall form by using the
-wrong fanout for a rune.  For example, `%+` ([**cenlus**](@/docs/hoon/hoon-expressions/rune/cen.md#cenlus),
-a function call whose sample is a cell) has three subhoons:
+wrong fanout for a rune.  For example, `%+` ([**cenlus**](@/docs/reference/hoon-expressions/rune/cen.md#cenlus),
+a function call whose sample is a cell) has three subreferences:
 
-```hoon
+```reference
```

Comment:
This is just one example, but a lot of mentions of the word 'hoon' are now 'reference'. Not just the syntax highlight metadata. Try the find/replace again with `/docs/hoon` -> `/docs/reference/`, etc.